ur_gazebo/launch/inc/load_ur.launch.xml

```diff
@@ -19,25 +19,39 @@
   -->
 
   <!--Parameter files -->
-  <arg name="joint_limit_params" doc="YAML file containing the joint limit values"/>
-  <arg name="kinematics_params" doc="YAML file containing the robot's kinematic parameters. These will be different for each robot as they contain the robot's calibration."/>
-  <arg name="physical_params" doc="YAML file containing the phycical parameters of the robots"/>
-  <arg name="visual_params" doc="YAML file containing the visual model of the robots"/>
+  <arg name="joint_limit_params" doc="YAML file containing the joint limit values" />
+  <arg name="kinematics_params" doc="YAML file containing the robot's kinematic parameters. These will be different for each robot as they contain the robot's calibration." />
+  <arg name="physical_params" doc="YAML file containing the phycical parameters of the robots" />
+  <arg name="visual_params" doc="YAML file containing the visual model of the robots" />
 
   <!--Common parameters  -->
-  <arg name="transmission_hw_interface" default="hardware_interface/PositionJointInterface" doc="The hardware_interface to expose for each joint in the simulated robot (one of: [PositionJointInterface, VelocityJointInterface, EffortJointInterface])"/>
-  <arg name="safety_limits" default="false" doc="If True, enable the safety limits controller"/>
+  <arg name="transmission_hw_interface" default="hardware_interface/PositionJointInterface" doc="The hardware_interface to expose for each joint in the simulated robot (one of: [PositionJointInterface, VelocityJointInterface, EffortJointInterface])" />
+  <arg name="safety_limits" default="false" doc="If True, enable the safety limits controller" />
   <arg name="safety_pos_margin" default="0.15" doc="The lower/upper limits in the safety controller" />
   <arg name="safety_k_position" default="20" doc="Used to set k position in the safety controller" />
 
-  <param name="robot_description" command="$(find xacro)/xacro '$(find ur_gazebo)/urdf/ur.xacro'
-    joint_limit_params:=$(arg joint_limit_params)
-    kinematics_params:=$(arg kinematics_params)
-    physical_params:=$(arg physical_params)
-    visual_params:=$(arg visual_params)
-    transmission_hw_interface:=$(arg transmission_hw_interface)
-    safety_limits:=$(arg safety_limits)
-    safety_pos_margin:=$(arg safety_pos_margin)
-    safety_k_position:=$(arg safety_k_position)"
-    />
+  <!-- FT sensor choice -->
+  <arg name="use_ft" />
+  <group unless="$(arg use_ft)">
+    <param name="robot_description" command="$(find xacro)/xacro '$(find ur_gazebo)/urdf/ur.xacro'
+  joint_limit_params:=$(arg joint_limit_params)
```

Comment:
Actually, before reformatting, as the contents are repeating in both of the `command` for `use_ft` true or false cases, I would create arguements for the string part that repeats in both cases and combine the strings like here: https://answers.ros.org/question/267043/how-to-combine-string-args-in-launch-file/

Less lines, less complexity ...